api/src/opentrons/protocol_engine/state/pipettes.py

```diff
@@ -220,6 +225,54 @@ def _update_current_well(self, command: Command) -> None:
             ):
                 self._state.current_well = None
 
+    def _update_deck_point(self, command: Command) -> None:
+        if isinstance(
+            command.result,
+            (
+                MoveToWellResult,
+                MoveToCoordinatesResult,
+                MoveRelativeResult,
+                PickUpTipResult,
+                DropTipResult,
+                AspirateResult,
+                DispenseResult,
+                BlowOutResult,
+                TouchTipResult,
+            ),
+        ):
```

Comment:
Could this be replaced by the destination mixin? The type checker might not like it, though

```suggestion
        if isinstance(command.result, DestinationPositionResult):
```